helm-chart/README.md

```diff
@@ -1,58 +1,88 @@
 # FAQ
+
 ## What should be configured before installation?
+
 1. You should have configured connection to existed k8s cluster
 2. Helm must be installed
 3. You should download chart external dependencies, using following commands:
-```
-   helm repo add bitnami https://charts.bitnami.com/bitnami
-   helm repo update
-   helm dependency update
-```
+
+    ```shell
+    helm repo add bitnami <https://charts.bitnami.com/bitnami>
+    helm repo update
+    helm dependency update
+    ```
+
 4. (Optional) Install ingress of your choice (for example: <https://github.com/kubernetes/ingress-nginx>)
 5. (Optional) Create certificates for https (for example: <https://github.com/jetstack/cert-manager/>)
 6. (Optional) Create values.override.yaml and override there parameters you want
 7. Change postgresql password as described below
-8. Add ingress to values.override.yaml(example also below)
-7. Deploy cvat using command below
+8. Create a rules.tar.gz archive containing at least all rego files in /cvat/apps/iam/rules not containing test in their name
```

Comment:
Could you add a shell command for this? Also please add where this archive should be placed.